client/meter/.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+    "env": {
+        "es6": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 2017
+    },
+    "rules": {
+        "indent": [
+            "error",
+            2
+        ],
+        "linebreak-style": [
+            "error",
+            "unix"
+        ],
+        "quotes": [
+            "error",
+            "single"
```

Comment:
I actually prefer double, is this a JS convention?

Reply:
It's a choice

Reply:
The use of quotes are inconsistent in the codebase. The choice here is rather arbitrary